microshift_welcome/index.adoc

```diff
@@ -0,0 +1,25 @@
+:_content-type: ASSEMBLY
+[id="microshift-welcome-index"]
+= {product-title} {product-version} Documentation
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-welcome-index
+
+[.lead]
+Welcome to the official {product-title} {product-version} documentation, where you can learn about {product-title} and start exploring its features.
+
+To navigate the {product-title} {product-version} documentation, use one of the following methods:
```

Comment:
```suggestion
To browse the {product-title} {product-version} documentation, use one of the following methods:
```
See the ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#m

Reply:
@tmalove Thanks, Tami. That's straight off the OCP welcome page. Can we put a ticket in for a change there in some backlog? I don't know where, do you?